### DISCUSSION TITLE
LANG: unbound variable with all script LXC

### DISCUSSION THREAD

**Discussion by Bisbico045** [tteck/Proxmox#3259] [Help Needed] [answered]
Hi,

I have a problem when i install Scripts LXC. I have the same error with all script LXC. The error is 
`"Setting up Container OS   /dev/stdin: line 89: LANG: unbound variable"`
Could you help to find the problem, plz ?

**Comment by tteck** [comment]
Using SSH?

**Comment by Bisbico045** [comment]
Yes i use terminal Webmin & SSH via Windows Terminal !

**Reply by tteck** [reply, marked as answer]
Use the Proxmox shell. You were warned about using SSH, but ignored.
https://github.com/tteck/Proxmox/blob/5d0e18634fa29240d10f1a76a3393e1f0537c708/misc/build.func#L121-L132
